app/backend/app.py

```diff
@@ -117,12 +120,20 @@ def ensure_openai_token():
 
 
 def create_app():
+    if APPLICATIONINSIGHTS_CONNECTION_STRING:
```

Comment:
Do you want to be more specific and say is not None, or do you intentionally want to account for falsy values like ""?

Reply:
Ah based on seeing Bicep, I think you're intentionally accounting for "" since thats whats get set by default.

Reply:
I want a False-y statement, if its an empty string it'll fail to boot